internal/platform/cpuid_arm64.go

```diff
@@ -0,0 +1,76 @@
+//go:build gc
+
+package platform
+
+import "runtime"
+
+// CpuFeatures exposes the capabilities for this CPU, queried via the Has, HasExtra methods.
+var CpuFeatures = loadCpuFeatureFlags()
+
+// cpuFeatureFlags implements CpuFeatureFlags interface.
+type cpuFeatureFlags struct {
+	isar0 uint64
+	isar1 uint64
+}
+
+// implemented in cpuid_arm64.s
+func getisar0() uint64
+
+// implemented in cpuid_arm64.s
+func getisar1() uint64
+
+func loadCpuFeatureFlags() CpuFeatureFlags {
+	switch runtime.GOOS {
+	case "darwin", "windows":
+		// These OSes require ARMv8.1, which includes atomic instructions.
```

Comment:
i get why darwin requires v8.1 by default (m series is so), but not sure about windows here. any reference or pointer?

Reply:
Windows 10 does not in fact require them, is it good enough to document this?
At least this change should not be a regression.